### PR TITLE
menteereport

### PR DESCRIPTION
**Pull request by GmeghnaRao**

### CODE REVIEW

File: src/test/java/org/shikshalokam/backend/TestMenteeReport.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                .header("X-auth-token", "bearer " + X_AUTH_TOKEN)
+                .get(endpoint);
+        logger.info("Response Body:\n" + responce.asPrettyString());
+        assertEquals(responce.getStatusCode(), 200, "Status code failed for fetching Mentee report");
+        int totalSessionsEnrolled = responce.getBody().jsonPath().getInt("result.total_session_enrolled");
+        System.out.println(responce.getBody().jsonPath().getInt("result.total_session_attended"));
+        int totalSessionsAttended = responce.getBody().jsonPath().getInt("result.total_session_attended");
+        assertEquals(totalSessionsEnrolled, 2, "Total sessions enrolled not valid");
+        assertEquals(totalSessionsAttended, 0, "Total sessions attended not valid");
+//        assertEquals(totalSessionsEnrolled, 2, "Total sessions enrolled not valid");
```
Comment (infa-rvarahag) [inline src/test/java/org/shikshalokam/backend/TestMenteeReport.java]: remove this commented code

File: src/test/java/org/shikshalokam/backend/TestMenteeReport.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
+        }
+        Response responce = given()
+                .header("X-auth-token", "bearer " + X_AUTH_TOKEN)
+                .get(endpoint);
+        logger.info("Response Body:\n" + responce.asPrettyString());
+        assertEquals(responce.getStatusCode(), 200, "Status code failed for fetching Mentee report");
+        int totalSessionsEnrolled = responce.getBody().jsonPath().getInt("result.total_session_enrolled");
+        System.out.println(responce.getBody().jsonPath().getInt("result.total_session_attended"));
```
Comment (infa-rvarahag) [inline src/test/java/org/shikshalokam/backend/TestMenteeReport.java]: remove the sysout statements

**Review by infa-rvarahag — CHANGES_REQUESTED** [review]
Please add your Testcase into new Suite file "MentorED_QA_API.xml"   
take the sample of any existing suite files.
run locally from the  mvn clean install with this suite file

**Comment by GmeghnaRao** [comment]
Required changes are done. Please review. New PR has been raised.

**Pull request closed without merging**